includes/class-sensei-quiz.php

```diff
@@ -1222,7 +1222,7 @@ public static function start_quiz_questions_loop() {
 		$sensei_question_loop['total']     = 0;
 		$sensei_question_loop['questions'] = array();
 
-		$questions = Sensei()->lesson->lesson_quiz_questions( get_the_ID() );
+		$questions = Sensei()->lesson->lesson_quiz_questions( get_the_ID(), 'publish' );
```

Comment:
It's not totally necessary, but it can avoid a `draft` question appearing in the quiz in an edge case.
But if the user is checking their previous answers, or it's being graded, the question will appear.